### ISSUE TITLE
Time embedding is near-constant; the network cannot distinguish timesteps

### ISSUE THREAD

**Issue by nikhilr612** [nikhilr612/tinyflow#7]
## Summary

`UNet.sinusoidal_embeddings` never oscillates, so the network can barely tell which timestep
it is denoising.

## Detail

```python
freqs = t * jax.numpy.geomspace(1, 1 / 10_000, num=self.time_embedding_dim // 2)
return jax.numpy.concat([jax.numpy.sin(freqs), jax.numpy.cos(freqs)])
```

With `t` in `[0, 1]` and frequencies running from `1` down to `1e-4`, every argument to
sin/cos lies in `[0, 1]`. Nothing completes even a fraction of a cycle, so the embedding is a
near-constant, monotone, almost-linear function of `t`.

Measured at `time_embedding_dim=128`:

| | current | with `t` scaled by 1000 |
|---|---|---|
| dims with std < 1e-3 across `t` | **71 / 128** | 0 / 128 |
| mean per-dim std across `t` | 0.020 | 0.49 |
| cos-similarity `emb(0.0)` vs `emb(1.0)` | **0.971** | 0.093 |

`emb(0)` and `emb(1)` being 0.97-parallel means the two ends of the schedule are nearly
indistinguishable to the network.

## Why it matters beyond the obvious

This is what makes the identity map in #6 the *global optimum* rather than merely an
attractor — a t-blind model must commit to a single function of `x_t`, and under the current
loss weighting the optimal such function is exactly `x_hat = x_t`.

## Proposed fix

Scale `t` into the standard DDPM range so the top frequency oscillates across the unit
interval:

```python
freqs = (self.time_scale * t) * jax.numpy.geomspace(
    1, 1 / 10_000, num=self.time_embedding_dim // 2
)
```

with `time_scale = 1000.0` as a module field. Additionally add the conventional 2-layer MLP
(`Linear -> silu -> Linear`) on top of the raw sinusoids, applied once in `__call__` before the
embedding reaches the blocks.

Note: do **not** put `time_scale` into `hparams` — `ImageFM.load` coerces every hparam with
`int()`, so a float would silently truncate.

**Comment by nikhilr612** [comment]
Fixed by `UNet.sinusoidal_embeddings` in models/unet.py (commit 7ad2eed). The frequency range now completes multiple cycles across `t in [0,1]`; embedding is no longer near-constant. Verified against current code.
